client/app/cloud/components/project/billing/instance-list/billing-instance-list.component.html

```diff
@@ -1,94 +1,72 @@
-                    <span data-ng-bind=" ::  instanceConsumption.instanceName"></span>
-                </td>
-                <td class="oui-datagrid__cell" data-title="{{ :: $ctrl.instanceColTotal | translate }}">
-                    <span data-ng-bind=" :: instanceConsumption.total"></span>
-                </td>
-                <td data-ng-if="$ctrl.showSwitchToMonthlyBillingOption || $ctrl.showAdditionalInstanceDetails" class="oui-datagrid__cell text-right no-space vertical-middle right-space-p4">
-                    <span data-ng-if="instanceConsumption.monthlyBilling && instanceConsumption.monthlyBilling.status === 'ok'"
-                            data-translate="cpbc_hourly_instance_monthly_billing_since"
-                            data-translate-values="{ since : (instanceConsumption.monthlyBilling.since | date:'shortDate') }">
-                    </span>
-                    <span data-ng-if="instanceConsumption.monthlyBilling && instanceConsumption.monthlyBilling.status === 'activationPending'">
-                        {{ :: 'cpbc_hourly_instance_monthly_billing_activationPending' | translate }}
-                        <i class="fa fa-clock-o left-space-m8"></i>
-                    </span>
-                    <span class="opacity6" data-ng-if="!instanceConsumption.monthlyBilling && instanceConsumption.isDeleted" data-translate="cpbc_hourly_instance_deleted"></span>
+<oui-datagrid data-rows="$ctrl.instanceConsumptionDetails" data-ng-if="$ctrl.instanceConsumptionDetails.length > 0">
+    <oui-column data-title=":: $ctrl.instanceColName | translate" data-sortable data-property="instanceName">
+        <span data-ng-bind=":: $row.instanceName"
+              data-responsive-popover="'app/cloud/components/project/billing/details-popover/instance-details-popover.html'"
+              data-popover-class="details-popover"
+              data-popover-is-open="$ctrl.DetailsPopoverService.isCurrentDetails($row)"
+              data-popover-placement="top"
+              data-popover-append-to-body="true"
+              data-popover-trigger="none"
+              data-ng-click="$ctrl.DetailsPopoverService.setCurrentDetails($row)"></span>
+    </oui-column>
+    <oui-column data-title=":: $ctrl.instanceColTotal | translate" data-property="total" data-sortable>
+    </oui-column>
+    <oui-column data-ng-if="$ctrl.showSwitchToMonthlyBillingOption || $ctrl.showAdditionalInstanceDetails">
+        <span data-ng-if="$row.monthlyBilling && $row.monthlyBilling.status === 'ok'"
+              data-translate="cpbc_hourly_instance_monthly_billing_since"
+              data-translate-values="{ since : ($row.monthlyBilling.since | date:'shortDate') }">
+        </span>
+        <span data-ng-if="$row.monthlyBilling && $row.monthlyBilling.status === 'activationPending'">
+            {{ :: 'cpbc_hourly_instance_monthly_billing_activationPending' | translate }}
```

Comment:
Thanks for having replaced tables with [<oui-datagrid>](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/datagrid).

Could you please also prefer using `ngBind` over `{{`?